### PR TITLE
Add Oneclick Mall and Full Transaction

### PR DESCRIPTION
**Pull request by a1b4**
Add Rest support for:
- Oneclick Mall
- Full Transaction

### CODE REVIEW

File: docker-unit-test/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+**Linux:**
+
+Instalar [Docker](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/) y [Docker-compose](https://docs.docker.com/compose/install/#install-compose).
+
+### Como usar
+
+Esta imagin de docker sirve para correr y comprobar la compatibilidad del sdk con distintas versiones de python
+
+**NOTA:** La primera vez que se ejecuta ./docker-build.sh demorará en instalar todo, esperar al menos unos 5 minutos.
```
Comment (iescaida) [inline docker-unit-test/README.md]: ups, se me paso esto aqui, deberia ser ./build.sh

Comment (a1b4) [reply docker-unit-test/README.md]: done! :)

Comment (iescaida) [reply docker-unit-test/README.md]: you were faster than me comment

File: docker-unit-test/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Instalar [Docker](https://docs.docker.com/docker-for-windows/install/), [Docker-compose](https://docs.docker.com/compose/install/#install-compose)
+
+**Linux:**
+
+Instalar [Docker](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/) y [Docker-compose](https://docs.docker.com/compose/install/#install-compose).
+
+### Como usar
+
+Esta imagin de docker sirve para correr y comprobar la compatibilidad del sdk con distintas versiones de python
```
Comment (gonzunigad) [inline docker-unit-test/README.md]:
```suggestion
Esta imagEn de docker sirve para correr y comprobar la compatibilidad del SDK con distintas versiones de python
```

**Pull request merged**